src/components/AuthStoreSetter.ts

```diff
@@ -1,21 +1,26 @@
 import _ from 'lodash/fp';
 import { useEffect } from 'react';
 import { AuthContextProps, useAuth } from 'react-oidc-context';
-import { loadAuthToken, OidcUser, processUser } from 'src/libs/auth';
+import { doUserLoaded, loadAuthToken, OidcUser } from 'src/libs/auth';
 import { useOnMount } from 'src/libs/react-utils';
-import { authStore } from 'src/libs/state';
+import { oidcStore } from 'src/libs/state';
 
 function AuthStoreSetter(): null {
+  // When the AuthContext changes, this component will reload
   const auth: AuthContextProps = useAuth();
 
   useOnMount(() => {
-    authStore.update(_.set(['authContext'], auth));
+    oidcStore.update(_.set(['authContext'], auth));
   });
   useEffect((): (() => void) => {
     const cleanupFns = [
-      auth.events.addUserLoaded((user: OidcUser) => processUser(user, true)),
-      auth.events.addUserUnloaded(() => processUser(null, false)),
-      auth.events.addAccessTokenExpired(() => {
+      // add{EVENT_NAME} will add this callback to this event
+      // ex: will add doUserLoaded to the userLoadedEvent
+      // load event called in the following fns https://github.com/authts/oidc-client-ts/blob/main/src/UserManager.ts
```

Comment:
I think it might be more helpful to link to documentation instead of code.

```suggestion
      // Subscribe to UserManager events.
      // For details of each event, see https://authts.github.io/oidc-client-ts/classes/UserManagerEvents.html
```